src/jit/compiler.h

```diff
@@ -303,6 +303,7 @@ class  LclVarDsc
     unsigned char       lvUsedInSIMDIntrinsic :1; // This tells lclvar is used for simd intrinsic
 #endif // FEATURE_SIMD
     unsigned char       lvRegStruct      :1;     // This is a reg-sized non-field-addressed struct.
+    unsigned char       lvIsVarAddrOnStack : 1;  // This contains var address on stack.
```

Comment:
It might be good to call this "lvIsVarAddrTmpOnStack" - in general it would be surprising to have a property (flag) on a lclVar that asserts something about its value (given that lclVars may have multiple assignments). However, in the case of a compiler-generated tmp, it makes more sense.